src/client/components/pages/entity-revisions.js

```diff
@@ -24,14 +26,28 @@ const Col = bootstrap.Col;
 const ListGroup = bootstrap.ListGroup;
 const ListGroupItem = bootstrap.ListGroupItem;
 
+/**
+* This is an extended class of the React Component to
+* render the 'Entity Revisions' page.
+*/
 class EntityRevisions extends React.Component {
 
+/**
+* Binds the class methods to their respective data.
+* @constructor
+* @param {object} props - Object of properties passed in the constructor
+*/
 	constructor(props) {
 		super(props);
 		this.renderHeader = this.renderHeader.bind(this);
 		this.renderRevision = this.renderRevision.bind(this);
 	}
 
+/**
+* Renders the Header of the page consisting of Revision ID and
+* entity information in the form of a table.
```

Comment:
This isn't quite correct - there is no table involved - only the entity name and disambiguation comment are rendered here.